entrees/nasi-goreng

```diff
@@ -0,0 +1,48 @@
+# Nasi goreng
+
+This recipe is based off [this Food Network recipe](https://www.foodnetwork.com/applink/recipes/food-network-kitchen/nasi-goreng-5482485). 
```

Comment:
It might be good to add a description of what Nasi goreng is.

Reply:
Added